README.md

```diff
@@ -9,12 +9,22 @@
 [![docs](https://img.shields.io/badge/docs-pkgdown-blue.svg)](https://icbi-lab.github.io/immunedeconv)
 
 ## Basic usage
+
+Deconvolution of human data:
+
 ```R
 immunedeconv::deconvolute(gene_expression_matrix, "quantiseq")
 ```
 
+Deconvolution of mouse data:
+
+```R
+immunedeconv::deconvolute_mouse(gene_expression_matrix, "quantiseq")
+```
+
 where `gene_expression_matrix` is a matrix with genes in rows and samples in columns. The rownames must be
-[HGNC](https://www.genenames.org/) symbols and the colnames must be sample names. The method can be one of
+[HGNC](https://www.genenames.org/) symbols and the colnames must be sample names. For human data, the method can be one of
```

Comment:
for mouse, it's [MGI](http://www.informatics.jax.org/mgihome/nomen/) gene symbols, I guess?